modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -22,21 +22,107 @@
 #include <algorithm>
 #include <limits>
 
+/**
+ * index sort functor
+ */
+template<typename T>
+class index_sorter {
+public:
+    index_sorter(T const & c) :
+        c(c)
+    {}
```

Comment:
weird indentation (also I thought our style guide explicitly states that `{` and `}` should be on separate lines - there is another of those further up)